### PR TITLE
Added Pause on Standby via CEC

### PR DESCRIPTION
**Pull request by filimonic**
Added Pause on Standby via CEC.
Simulateous commit ( pull reqest 109 ) to libcec project

### CODE REVIEW

**Comment by Jalle19** [comment]
Ping @opdenkamp

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -16619,3 +16619,10 @@ msgstr ""
 msgctxt "#38016"
 msgid "%d fps"
 msgstr ""
+
+#. Label for a select option
+#: xbmc/peripherals/devices/PeripheralCecAdapter.cpp
+#: system/peripherals.xml
+msgctxt "#38017"
+msgid "Pause"
```
Comment (da-anda) [inline addons/resource.language.en_gb/resources/strings.po]: I think "pause playback" is a bit more userfriendly

**Comment by Montellese** [comment]
Why did you commit `project/BuildDependencies/include/libcec/cectypes.h`? All files under `project/BuildDependencies/include/` are automatically downloaded and must not be checked into git.

**Comment by opdenkamp** [comment]
changes to libCEC should indeed no be done here, but in libCEC itself. then the version has to be bumped and checked here before using it.

but the idea once was that these settings could be shared across apps using libCEC. which is unlikely to happen any time soon ;-) so instead of doing all this kind of stuff just to add a setting, I think we should remove all the bits stored in the configuration struct and only store it locally and check that instead.

**Comment by Jalle19** [comment]
@filimonic while you're at it, would you mind adding a "Stop playback" option as well? It's been an often requested feature on the forums, arguable more requested than pause playback.

**Comment by filimonic** [comment]
will rewrite like @opdenkamp said

**Pull request closed without merging**